docs/gettingstarted/index.md

```diff
@@ -1,60 +1,119 @@
 Getting Started with Forge
 ==========================
 
-This is a simple guide to get you from nothing to a basic mod. The rest of this documentation is about where to go from here.
+If you have never made a Forge mod before, this section will provide the minimum amount of information needed to setup a Forge development environment. The rest of the documentation is about where to go from here.
+
+Prerequisites
+-------------
+
+* An installation of the Java 17 Development Kit (JDK) and 64-bit Java Virtual Machine (JVM). Forge recommends and officially supports [Eclipse Temurin][jdk].
+
+!!! warning
+    Make sure you are using a 64-bit JVM. One way of checking is to run `java -version` in a terminal. Using a 32-bit JVM will cause some problems when using [ForgeGradle].
+
+* Familiarity with an Integrated Development Environment (IDE).
+    * It is recommended to use an IDE with Gradle integration.
 
 From Zero to Modding
 --------------------
 
-1. Obtain a Java 17 Development Kit (JDK) and a 64-bit Java Virtual Machine (JVM). Minecraft and MinecraftForge both compile against Java 17 and as such should be used for development. Using a 32-bit JVM will result in some problems when running the below gradle tasks. You can obtain one from [Eclipse Adoptium][jdk].
-2. Obtain the Mod Development Kit (MDK) from Forge's [files][] site.
-3. Extract the downloaded MDK into an empty directory. You should see a bunch of files along with an example mod placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects. These files are:
+1. Download the Mod Developer Kit (MDK) from the [Forge file site][files] by clicking 'Mdk' followed by the 'Skip' button in the top right after waiting for a period of time. It is recommended to download the latest version of Forge whenever possible.
+1. Extract the downloaded MDK into an empty directory. This will be your mod's directory, which should now contain some gradle files and a `src` subdirectory containing the example mod.
+
+!!! note
```

Comment:
Same comment as above with regards to indenting adomitions that form a 'part' of a list item. (I can't leave a suggestion for the whole admonition because of the removals in the way.)